layersvt/screenshot.cpp

```diff
@@ -167,8 +168,10 @@ typedef struct {
     std::list<VkCommandPool> commandPools;
```

Comment:
Since we now create our own command pool, the variable commandPools should be removed, as well as all references to it. Also remove the CreateCommandPools hook function and all references to it.